### DISCUSSION TITLE
Publish FOSS version of Ente Photos on Github.

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#3060] [Enhancements]
Currently the only method of getting a FOSS build of Ente Photos is to use F-Droid. This is not ideal for people who prefer to get the APK directly from the developer and would rather avoid F-Droid. This is a feature request for publishing a FOSS build of Ente Photos on Github, without any proprietary libraries.

**Comment by ashilkn** [comment]
We do publish on Github. Checkout https://github.com/ente-io/ente/releases

**Reply by ghost** [reply]
Don't these APKs use the proprietary Google Billing SDK?

**Reply by ghost** [reply]
Firebase Messaging is also included in the Github builds of Ente Photos. My feature request is for a build without any proprietary Google libraries (Firebase, FCM, Google Billing) to be published on Github alongside the currently published APKs.

**Reply by ghost** [reply]
@ashilkn please reopen.

**Reply by ghost** [reply]
The discussion has been closed again, is this not planned by Ente currently?
